client/src/components/common/Location/index.js

```diff
@@ -1,40 +1,54 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Icon } from 'antd';
+import L from 'leaflet';
```

Comment:
please use descriptive name
```suggestion
import Leaflet from 'leaflet';
```

Reply:
Note: Accepting this suggestion will require additional updates elsewhere in the file where `L` is referenced

Reply:
it's used as in documentation, you can see doc https://leafletjs.com/